lib/currency.rb

```diff
@@ -0,0 +1,29 @@
+module Currency
+  extend self
+  
+  def args(*currency)
+    currency = currency.flatten.compact
+    return if currency.blank?
+    case currency.length
+    when 0
+      nil
```

Comment:
Unreachable after `blank?` ⇑

Hopefully the existence case-branch permits to just remove line 6.